libraries/core-react/src/Menu/Menu.test.jsx

```diff
@@ -26,8 +26,9 @@ describe('Menu', () => {
     expect(menuContainer).toHaveStyleRule('background', 'red')
   })
   it('is visible when open is true & anchorEl is set', () => {
+    const handleOnClose = jest.fn()
     render(
-      <Menu open>
+      <Menu open onClose={handleOnClose}>
```

Comment:
remove the new `onClose` functions in tests. onClose is optional so should be tested accordingly